crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1799,3 +1790,197 @@ pub struct DisputeObj {
     pub dispute_id: String,
     pub status: String,
 }
+
+#[cfg(test)]
+mod test_validate_shipping_address_against_payment_method {
+    use api_models::enums::CountryCode;
+    use masking::Secret;
+
+    use crate::{
+        connector::stripe::transformers::{
+            validate_shipping_address_against_payment_method, StripePaymentMethodType,
+            StripeShippingAddress,
+        },
+        core::errors,
+    };
+
+    #[test]
+    fn should_return_ok() {
+        // Arrange
+        let stripe_shipping_address = create_stripe_shipping_address(
+            Some("name".to_string()),
+            Some("line1".to_string()),
+            Some(CountryCode::AD),
+            Some("zip".to_string()),
+        );
+
+        let payment_method = &StripePaymentMethodType::AfterpayClearpay;
+
+        //Act
+        let result = validate_shipping_address_against_payment_method(
+            &stripe_shipping_address,
+            payment_method,
+        );
+
+        // Assert
+        assert!(result.is_ok());
+    }
+
+    #[allow(clippy::unwrap_used)]
```

Comment:
You can add this annotation to the beginning of the tests module here like so:

```rust
#[cfg(test)]
mod test_validate_shipping_address_against_payment_method {
    #![allow(clippy::unwrap_used)]
    // ...
}
```

Also, remove the annotations you've added for individual tests.